101-workshop-nodejs/ex0-what-is-serverless/serverless-defined.md

```diff
@@ -38,10 +30,4 @@ _adjective_ (of computing)
 1. _Event-driven application programming using stand-alone functions with no deployment or operational considerations_
 {% endhint %}
 
-### Serverless "is a" misnomer
-
-To be clear...
-
-{% hint style="success" %}
```

Comment:
I like ending sections with an HTML extension box that captures/highlights the definitive concept we want to be completely clear about going forward.

Reply:
I don't dislike like extension boxes. There were three on this page, which (to me) made it difficult to tell exactly what was **most** important (since so many things were in boxes). If you'd like to keep the last one, maybe the serverless definition doesn't go in a box but is just  shown as normal text? And then we'd have to decide on a better "header" than "To be clear..." Maybe we could do a wrap up of the whole page.. something like:

- Classic servers **are** needed to run your code.
- There are many serverless ideals, but we'll only be adhering to one definition. 
- To be clear, servers exist but are not a concern for developers!

And use the "observations" heading we'd discussed incorporating. Just trying to think of uniformity when creating/using these extension boxes.